message_store_map_switcher Does not compile

Message store Map Switcher is not compiling anymore, it looks like a mongodb version problem, full log can be fpund [here](https://lcas.lincoln.ac.uk/buildfarm/job/Kpr__strands_navigation__ubuntu_xenial_amd64/1/console), relevant bits as follows:
```sh
22:21:56 [ 11%] Building CXX object CMakeFiles/message_store_map_saver.dir/src/message_store_map_saver.cpp.o
22:21:59 In file included from /usr/include/c++/5/atomic:38:0,
22:21:59                  from /opt/ros/kinetic/include/mongo/platform/atomic_word_cxx11.h:29,
22:21:59                  from /opt/ros/kinetic/include/mongo/platform/atomic_word.h:21,
22:21:59                  from /opt/ros/kinetic/include/mongo/base/status.h:23,
22:21:59                  from /opt/ros/kinetic/include/mongo/util/assert_util.h:23,
22:21:59                  from /opt/ros/kinetic/include/mongo/bson/bsontypes.h:20,
22:21:59                  from /opt/ros/kinetic/include/mongo/bson/bsonelement.h:25,
22:21:59                  from /opt/ros/kinetic/include/mongo/bson/bsonobj.h:27,
22:21:59                  from /opt/ros/kinetic/include/mongo/client/bulk_operation_builder.h:20,
22:21:59                  from /opt/ros/kinetic/include/mongo/client/dbclientinterface.h:29,
22:21:59                  from /opt/ros/kinetic/include/mongo/client/dbclient_rs.h:23,
22:21:59                  from /opt/ros/kinetic/include/mongo/client/dbclient.h:59,
22:21:59                  from /opt/ros/kinetic/include/mongodb_store/message_store.h:17,
22:21:59                  from /tmp/catkin_workspace/src/strands_navigation/message_store_map_switcher/src/message_store_map_saver.cpp:1:
22:21:59 /usr/include/c++/5/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
22:21:59  #error This file requires compiler and library support \
22:21:59   ^
22:22:00 In file included from /opt/ros/kinetic/include/mongo/base/data_cursor.h:21:0,
22:22:00                  from /opt/ros/kinetic/include/mongo/bson/bsonelement.h:24,
22:22:00                  from /opt/ros/kinetic/include/mongo/bson/bsonobj.h:27,
22:22:00                  from /opt/ros/kinetic/include/mongo/client/bulk_operation_builder.h:20,
22:22:00                  from /opt/ros/kinetic/include/mongo/client/dbclientinterface.h:29,
22:22:00                  from /opt/ros/kinetic/include/mongo/client/dbclient_rs.h:23,
22:22:00                  from /opt/ros/kinetic/include/mongo/client/dbclient.h:59,
22:22:00                  from /opt/ros/kinetic/include/mongodb_store/message_store.h:17,
22:22:00                  from /tmp/catkin_workspace/src/strands_navigation/message_store_map_switcher/src/message_store_map_saver.cpp:1:
22:22:00 /opt/ros/kinetic/include/mongo/base/data_view.h: In member function ‘const mongo::ConstDataView& mongo::ConstDataView::readNative(T*, size_t) const’:
22:22:00 /opt/ros/kinetic/include/mongo/base/data_view.h:43:23: error: ‘is_trivially_copyable’ is not a member of ‘std’
22:22:00          static_assert(std::is_trivially_copyable<T>::value,
22:22:00                        ^
22:22:00 /opt/ros/kinetic/include/mongo/base/data_view.h:43:51: error: expected primary-expression before ‘>’ token
...
22:22:01 <== Failed to process package 'message_store_map_switcher': 
22:22:01   Command '['/tmp/catkin_workspace/install_isolated/env.sh', 'make', '-j1']' returned non-zero exit status 2
22:22:01 
22:22:01 Reproduce this error by running:
22:22:01 ==> cd /tmp/catkin_workspace/build_isolated/message_store_map_switcher && /tmp/catkin_workspace/install_isolated/env.sh make -j1
22:22:01 
22:22:01 Command failed, exiting.

```

--- a/message_store_map_switcher/CMakeLists.txt
+++ b/message_store_map_switcher/CMakeLists.txt
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 2.8.3)
 project(message_store_map_switcher)
 
 
+add_compile_options(-std=c++11)
+
 find_package(catkin REQUIRED COMPONENTS
   mongodb_store
   map_server  

